src/app.js

```diff
@@ -10,10 +10,10 @@ app.use(express.urlencoded({ extended: false }));
 app.use(express.json());
 
 if (process.env.NODE_ENV !== 'test') {
-  app.listen(port, () => {
-    console.log(`Server listening on port ${port}`);
-    connectToMongo();
-  });
+    app.listen(port, () => {
+        // console.log(`Server listening on port ${port}`);
```

Comment:
why is this commented?

Reply:
@halitbatur, the linter was giving me warnings not to use console logs. I thought that we can use a logger instead.